src/pane-element.coffee

```diff
@@ -37,8 +38,21 @@ class PaneElement extends HTMLElement
     handleBlur = (event) =>
       @model.blur() unless @contains(event.relatedTarget)
 
+    handleDragOver = (event) =>
+      event.preventDefault()
+      event.stopPropagation()
+
+    handleDrop = (event) =>
+      event.preventDefault()
+      event.stopPropagation()
+      @getModel().activate()
+      pathsToOpen = _.pluck(event.dataTransfer.files, 'path')
```

Comment:
Now I have removed the dependency to _ and replaced with Array::map.
```javascript
pathsToOpen = Array::map.call event.dataTransfer.files, (file) -> file.path
```
Thank you.